app/views/wishlist_items/_wishlist_item.html.erb

```diff
@@ -23,15 +23,15 @@
 
         <div class="row">
           <!-- Staff Message -->
-          <% if wishlist_item.staff_message %>
-            <p>
-            <strong>Staff Message:</strong>
-            <span class="card-text"><%= wishlist_item.staff_message %></span>
+          <% if wishlist_item.staff_message && wishlist_item.staff_message.length > 0 %>
+            <p class="staff-message">
```

Comment:
Would this work if you just make it a `div`? And that way you won't need the `width: 100%` scss change?

Reply:
Not sure. I will test it out now.

Reply:
So changing just staff message from a `<p>` to a `<div>` looks like:
![screen shot 2017-12-05 at 5 58 48 pm](https://user-images.githubusercontent.com/8367411/33635465-11178486-d9e6-11e7-85bc-5906e5dbe43b.png)

I personally don't like it cause the staff message then loses the default styles of the p tag so we'd have to add some style for margin anyway. Also the item details and admin tools should also become divs then for consistency imo and I tried changing the others to a div and they all end up on one line so it would require similar styling anyway. 

Let me know whatcha think.

Reply:
I agree. Looks better with the `p` tag. Thanks for trying it out 😃